libs/components/lookup/src/assets/locales/resources_en_US.json

```diff
@@ -3,10 +3,22 @@
     "_description": "Label for autocomplete add button",
     "message": "New"
   },
+  "skyux_autocomplete_multiple_results": {
+    "_description": "Screen reader only text for multiple available results",
+    "message": "{0} results available."
+  },
   "skyux_autocomplete_no_results": {
     "_description": "Placeholder text for autocomplete input component when no results are found",
     "message": "No matches found"
   },
+  "skyux_autocomplete_one_result": {
+    "_description": "Screen reader only text for singular available result",
+    "message": "One result available."
+  },
+  "skyux_autocomplete_results": {
```

Comment:
This string isn't needed correct? Can it be removed just to not clutter things and to avoid unneeded translation costs in the future if we ever get these files translated?